src/kaocha/runner.clj

```diff
@@ -15,7 +15,7 @@
             [kaocha.output :as output]
             [kaocha.plugin :as plugin]
             [kaocha.result :as result]
-            [kaocha.specs :as specs]
+            #_[kaocha.specs :as specs] ;; required for some reason?
```

Comment:
If you require namespaces for side effect, you can leave out the alias and clj-kondo won't report it as unused.

Reply:
When namespaces are required for side-effects, the convention we use is to `(require '...)` them below the ns form.

Reply:
This also works (for clj-kondo)